spec/fixtures/conditionally_loaded_1.rb

```diff
@@ -0,0 +1,3 @@
+# some comment
+puts "wargh"
+puts "wargh 1"
```

Comment:
these files were just missing but referenced in tests and sicne we have code that removes missing files from saved results it caused problems so added them :)